### ISSUE TITLE
CSR sparse matrix returns 0-D array on selection (instead of scalar)

### ISSUE THREAD

**Issue by jakirkham** [scipy/scipy#10373]
In SciPy 1.3.0, it appears that selecting a value from a CSR sparse matrix (unsure if other sparse matrices are affected) now returns a 0-D array; whereas, in SciPy 1.2 (and before) it returned a scalar. Below is an example of this behavior.

### Reproducing code example:

```python
In [1]: import numpy as np

In [2]: import scipy.sparse as sp

In [3]: x = np.array([1, 2, 3])

In [4]: spx = sp.csr_matrix(x)

In [5]: spx[0, 0]  # previously returned a scalar
Out[5]: array(1)
```

### Scipy/Numpy/Python version information:

SciPy 1.3.0
NumPy 1.16.4

cc @stuartarchibald

**Comment by peterbell10** [comment]
Duplicate of #10206, fixed in #10207

**Comment by jakirkham** [comment]
Awesome! Sorry for the duplicate. Glad to hear it is already fixed. 😄

Closing...
